Kickstarter-iOS/DataSources/CommentRepliesDataSource.swift

```diff
@@ -30,14 +34,23 @@ internal final class CommentRepliesDataSource: ValueCellDataSource {
 
     self.clearValues(section: Section.replies.rawValue)
 
-    let newComments = comments.map { comment in
-      (comment, project)
+    let newReplies = replies.map { [project] reply in
```

Comment:
Not sure we need the capture list here for `project` because this isn't an async closure.

Reply:
Updated.